content/desktop/settings/windows.md

```diff
@@ -216,6 +212,25 @@ The HTTPS proxy settings used for scanning images are set using the `HTTPS_PROXY
 If you are running Windows containers in Docker, you can allow the Windows Docker daemon to use Docker Desktop's internal proxy, with the **Use proxy for Windows Docker daemon** setting.
 This is useful when a corporate proxy that requires authentication is manually configured or set at the system level. If you are an admin for your organization and have a Docker Business subscription, you can control this setting with [Settings management](../hardened-desktop/settings-management/configure.md) using the `windowsDockerdPort` parameter.
 
+#### Proxy authentication
+
+If the proxy requires authorization then Docker Desktop dynamically asks
+the developer for a username and password. All passwords are stored securely in the OS credential store.
+
+Docker Desktop supports Basic, Kerberos and NTLM proxy authentication methods. 
+
+##### Basic authentication
+
+If your proxy uses Basic authentication, Docker Desktop dynamically prompts developers for a username and password. 
+
+It's recommended that you use an `https:// URL` for HTTP/HTTPS proxies to protect passwords during network transit. Docker Desktop also supports TLS 1.3 for communication with proxies.
+
+##### Kerberos and NTLM authentication
+
+Kerberos and NTLM proxy authentication are available for Pro, Team, and Business subscribers with Docker Desktop version 4.30 and later. No additional configuration is needed beyond specifying the proxy IP address and port.
+
+You can enjoy a seamless experience without being interrupted by prompts for proxy credentials, reducing the risk of account lockouts due to incorrect sign in attempts.
```

Comment:
I'd rephrase this sentence to describe the why as well. i.e. with NTLM/Kerberos, the user isn't prompted for auth to the proxy because with these methods authentication is centralized (or something of that nature)